quickstarts/README.md

```diff
@@ -15,4 +15,4 @@ To override the JKube release you can pass the `jkube.version` property when run
 e.g.
 ```shell script
 $ mvn -Pkubernetes clean install k8s:build -Djkube.version=1.33.7
-```
\ No newline at end of file
+```
```

Comment:
Could you please revert changes done to this file? It's not clear in diff what changes have been made.
You should only keep files relevant to the issue you've linked in the pull request.